spec/rails_helper.rb

```diff
@@ -46,7 +49,7 @@
   end
   config.infer_spec_type_from_file_location!
   config.include ApplicationHelper
-  config.include Requests::JsonHelpers, type: :controller
+  config.include Requests::JsonHelper, type: :controller
```

Comment:
I am using it to add methods from the Request Helper file to my controller.
Did not want to do it in each file